source/preprocessor.tex

```diff
@@ -2219,20 +2219,20 @@
 \item
 \indextext{__file__@\mname{FILE}}%
 \mname{FILE}\\
-The presumed name of the current source file (a character string
-literal).
-\begin{footnote}
-The presumed source file name can be changed by the \tcode{\#line} directive.
-\end{footnote}
+A character string literal representing the presumed name of
+the current source file.
+\begin{note}
+The presumed source file name can be changed by the \tcode{\#line} directive\iref{cpp.line}.
+\end{note}
 
 \item
 \indextext{__line__@\mname{LINE}}%
 \mname{LINE}\\
-The presumed line number (within the current source file) of the current source line
-(an integer literal).
-\begin{footnote}
-The presumed line number can be changed by the \tcode{\#line} directive.
-\end{footnote}
+A decimal integer literal representing the presumed line number
```

Comment:
The incoming text says "an integer literal", allowing hex or binary literals.  You change that to "decimal", which feels like a normative change.